app/javascript/angular/code/services/_draw_session.js.erb

```diff
@@ -1,5 +1,12 @@
 import _ from "underscore";
-import { removeMarker } from "./google/_map";
+import { removeMarker } from "./google/_map.js.erb";
+
+<% helpers = ActionController::Base.helpers %>
+
+const locationMarker1Path = "<%= helpers.image_path('location_marker1.svg') %>";
+const locationMarker2Path = "<%= helpers.image_path('location_marker2.svg') %>";
+const locationMarker3Path = "<%= helpers.image_path('location_marker3.svg') %>";
+const locationMarker4Path = "<%= helpers.image_path('location_marker4.svg') %>";
```

Comment:
I wanted to extract a nice `imagePath` service to be used in nicely extracted functions but it's proven to complicate things too much. so I decided not to hate this.

Reply:
Complicate things how? Maybe we can investigate if there is a simpler solution to extract something here?

Reply:
I don't want to bother at this moment but might try doing it again sometime later when I touch the code again, and explain it better. might be that I was trying to do too much at once, but I extracted sth like this: 

```
const imagePath = imageName => {
  <% helpers = ActionController::Base.helpers %>  
  return "<%= helpers.image_path(imageName) %>";
};

export default imagePath;
```
and it wasn't enough just to use it in the particular files, things got broken. took too long so I wen't back to this simplistic version. I don't it's worth spending too much time on.